jobs/job-scheduler/src/main/java/com/paladincloud/jobscheduler/service/JobScheduler.java

```diff
@@ -97,33 +106,41 @@ public void scheduleCollectorJobs() {
 
         try {
             ConfigUtil.setConfigProperties();
+			  azureEnabled=Boolean.parseBoolean(System.getProperty(AZURE_ENABLED));
+			  gcpEnabled=Boolean.parseBoolean(System.getProperty(GCP_ENABLED));
+			  awsEnabled=Boolean.parseBoolean(System.getProperty(AWS_ENABLED)); 
+			  redHatEnabled=Boolean.parseBoolean(System.getProperty(REDHAT_ENABLED));
+
+				
+				if (awsEnabled) {
+					addCollectorEvent(putEventsRequestEntries, awsBusDetails);
+				}
+				if (azureEnabled) {
+					addCollectorEvent(putEventsRequestEntries, azureBusDetails);
+				}
+				 
+				if (gcpEnabled) {
```

Comment:
GCP should still go through the old flow

Reply:
yes @plyubich. I reverted the GCP changes.